eng/versioning/version_client.txt

```diff
@@ -81,6 +81,7 @@ com.microsoft.azure:spring-data-cosmosdb;3.0.0-beta.1;3.0.0-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
```

Comment:
Are these changes also tested on service bus? I have a feeling it might break it.